### ISSUE TITLE
HOWDOI_DISABLE_SSL=1 doesn't work

### ISSUE THREAD

**Issue by EdwardHXu** [gleitz/howdoi#160]
my howdoi failed to establish network connection all the time so I use Proxifier to figure out what the problem is. Then I find out that howdoi always try to use google.com:443 which is a https connection. And then I export HOWDOI_DISABLE_SSL=1 to disable https. However, it doesn't work at all. It firstly goes to port 80 while then turn to port 443 again...
![image](https://cloud.githubusercontent.com/assets/19581733/24087858/09539c48-0d5f-11e7-8da1-cdd53b2b80ca.png)
So, how can I work out this issue? Thanks!

**Comment by ruanima** [comment]
谷歌默认重定向到https了吧，所以HOWDOI_DISABLE_SSL=1没有用

**Comment by gleitz** [comment]
What version of howdoi are you using? Let me know with `howdoi -v`

**Comment by EdwardHXu** [comment]
1.1.9  thank you!

**Comment by gleitz** [comment]
I suspect it has something to do with that Singapore redirect. You might want to try downloading howdoi from source and changing [these lines](https://github.com/gleitz/howdoi/blob/HEAD/howdoi/howdoi.py#L47-L52) to use google.com.sg.

**Comment by EdwardHXu** [comment]
thanks @gleitz I'll try

**Comment by gleitz** [comment]
Closing for now. Please post if you have additional information.

**Comment by ttmdear** [comment]
I have same problem. I set HOWDOI_DISABLE_SSL=1, and error : 
Failed to establish network connection

**Comment by EdwardHXu** [comment]
@gleitz 
Still have not solved this problem... I've figured out this Singapore redirect but google still redirects to 443 port which is a https port after I have exported HOWDOI_DISABLE_SSL=1.
![temp](https://cloud.githubusercontent.com/assets/19581733/25984181/b92abd8c-371b-11e7-96dd-4624c4bbcf41.jpeg)
So I thought why I have to use Google as search engine instead of stackoverflow. I change these lines into these below:
![temp1](https://cloud.githubusercontent.com/assets/19581733/25984232/15727774-371c-11e7-85e6-eab0266476a3.jpeg)
but I got"Sorry, couldn't find any help with that topic"
So, how can I solve this issue caused by google443 or how can I use stackoverflow directly?
